packages/lib-react-components/src/Markdownz/Markdownz.stories.js

```diff
@@ -23,25 +75,38 @@ export default {
   }
 }
 
-export const Default = () => <Markdownz>{markdownExample}</Markdownz>
+export const Default = () => (
+  <>
+    <MarkdownExamplesTable />
+    <Markdownz>
+      {markdownExample}
+    </Markdownz>
+  </>
+)
 
 export const InProjectContext = () => (
-  <Markdownz projectSlug='zooniverse/snapshot-wakanda'>
-    {markdownExample}
-  </Markdownz>
+  <>
+    <MarkdownExamplesTable projectSlug='zooniverse/snapshot-wakanda' />
+    <Markdownz>
+      {markdownExample}
+    </Markdownz>
+  </>
 )
 
 export const WithCustomComponents = () => (
-  <Markdownz components={{ tr: TableRowWithBorder }}>
-    {markdownExample}
-  </Markdownz>
+  <>
+    <MarkdownExamplesTable components={{ tr: TableRowWithBorder }} />
+    <Markdownz>
+      {markdownExample}
+    </Markdownz>
+  </>
 )
 
 export const GridExample = () => (
```

Comment:
In this story, there's a video string: `![alt](https://static.zooniverse.org/www.zooniverse.org/assets/home-video.mp4 =300x)`, however the width constraint no longer works on this branch. Adding a width constraint to a markdown video was a bonus feature added to FEM, and is not supported in PFE. 

Examples:
- FEM: https://frontend.preview.zooniverse.org/projects/goplayoutside3/fem-test-project/about/research
- PFE: https://www.zooniverse.org/projects/goplayoutside3/fem-test-project/about/research

Thoughts on if we should keep the width constraint for videos in FEM? I don't know of any projects currently using it.

Reply:
🤷🏻‍♂️. It's not supported by the [html5-embed plugin ](https://www.npmjs.com/package/markdown-it-html5-embed)but we can write our own plugins if project teams want this. Should `<video>` use height and width attributes, or should the height and width be styled with CSS?

Reply:
I’m going to say it’s fine to **not** support video “resize in markdown” in FEM, and Zooniverse can recommend to teams that they upload example videos via panoptes rather than a service like youtube (see note below about communication to Zooniverse PMs and FEM project teams).

For example, I looked at the Manatee Chat video examples in their tutorial and field guide in [FEM](https://local.zooniverse.org:3000/projects/cetalingua/manatee-chat/classify/workflow/24994?env=production), and sizing looks great 👍

Reply:
Sounds good. The Remark implementation writes video and audio as HTML `<img>` tags, with height and width, then maps images to a React video component if `src` is a video URL.